pkg/storegateway/gateway_test.go

```diff
@@ -57,6 +57,13 @@ import (
 	"github.com/grafana/mimir/pkg/util/validation"
 )
 
+var (
+	// RingNumTokens is the number of tokens registered in the ring by each store-gateway
+	// instance. We use a safe default instead of exposing to config option to the user
+	// in order to simplify the config.
+	RingNumTokens = 512
```

Comment:
can we keep this as a constant and call it `RingNumTokensDefault`? This way we use it in the config and in the tests too

Reply:
ack